sailor/sap_iot/fetch.py

```diff
@@ -225,9 +225,7 @@ def get_indicator_data(start_date: Union[str, pd.Timestamp, datetime.timestamp,
 
                 results = pd.merge(results, data, on=['equipment_id', 'timestamp'], how='outer')
 
-        if request_ids:
-            print('Waiting for data export:')
-        else:
+        if not request_ids:
```

Comment:
oops I did not notice during review

btw
now you can have it even simpler and put this into the while expression -> (`while request_ids:`)